doc: Tiny fix (#2468)
cloudquery/cloudquery: website/pages/blog/cloudquery-v1-release.md

- **Built-in Concurrency** - High performance is critical--especially for infrastructure tasks—-this is why we take advantage of the excellent concurrency support in Go. We do smart scheduling for the source plugin requests, so plugins can extract information in parallel from multiple APIs as well as multiple accounts. This is a common use case, especially when syncing from the big cloud providers.
- **Structured logging** - ELT depends on third party APIs which can have down time, permissions errors, throttling errors and more. The key to running such pipelines successfully is monitoring and understanding errors. The new SDK provides structured logging with fast performance from the `zerolog` library, so it’s easy to understand errors, counts, timings, etc.

> - **Cross Platform / Cross Language support** - Plugins are now communicating via simple gRPC protocol which means plugins can be compiled to any OS and architecture and can be written in any language (although we are currently only supporting Go). Previously they also worked over via gRPC but we used HashiCorp go-plugin abstraction on top of gRPC which turned out to be not a great fit for us and unnecessary size and documentation bloat, the new dependency light CLI size was reduced from 40MB->21MB and made development and debugging a breeze.

Reviewer: ```suggestion
- **Cross Platform / Cross Language support** - Plugins are now communicating via simple gRPC protocol, which means plugins can be compiled to any OS and architecture and can be written in any language (although we are currently only supporting Go). Previously they also streamed over gRPC, but we used HashiCorp's go-plugin abstraction on top of gRPC, which turned out not to be a great fit for us, causing unnecessary size and documentation bloat. The new dependency-light CLI size is now 21MB, down from 40MB, and makes development and debugging a breeze.
```

Reviewer: A few more fixes :)
